### DISCUSSION TITLE
How can you remove the nav for the frontpage alone?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#3382] [Q&A] [answered]
Currently in the process of making a frontpage for the docs of my Discord bot.

Everything looks the way I want, except for one small thing, which is the top section.

In the top section would I like to get rid of the navigation tabs, since they are already linked to in the bottom section, the extra logo on the left and the search bar itself.

Thing is, I'm not sure what parts I can and cannot alter in the HTML files/partials to only change it for the frontpage but not the remainder of the docs.

Help for this would be appreciated.

- Current preview: https://deploy-preview-185--purrbot-site.netlify.app/
- Current PR: https://github.com/purrbot-site/Docs/pull/185
- frontpage.html file: https://github.com/purrbot-site/Docs/blob/feature/add-frontpage/theme/frontpage.html

**Comment by squidfunk** [comment, marked as answer]
The easiest way is to just use CSS in your Markdown file:

``` markdown
<style>
.md-tabs {
  display: none;
}
</style>
```

**Reply by Andre601** [reply]
Okay. Thanks.

There is already some existing style settings in the template. Is that anything to keep an eye on or would it not matter?

**Reply by squidfunk** [reply]
Shouldn’t matter.
